src/napari_imagej/_napari_converters.py

```diff
@@ -266,6 +266,25 @@ def _layer_to_roitree(layer: Shapes):
     return rois
 
 
+# -- Points / RealPointCollection -- #
+
+
+def _points_to_realpointcollection(points):
+    pts = [realPoint_from(x) for x in points.data]
+    ptList = ArrayList(pts)
+    return DefaultWritableRealPointCollection(ptList)
+
+
+def _realpointcollection_to_points(collection):
+    data = np.zeros((collection.size(), collection.numDimensions()))
+    tmp_arr_dims = int(collection.numDimensions())
+    tmp_arr = JArray(JDouble)(tmp_arr_dims)
+    for i, pt in enumerate(collection.points()):
+        pt.localize(tmp_arr)
+        data[i, :] = tmp_arr
+    return Points(data=data)
```

Comment:
Can we explain what is going on here? A docstring, or some comments, would go a long way.

Reply:
Some comments were added in 28c39dd